Source/GitReleaseManager.IntegrationTests/ReleaseNotesBuilderIntegrationTests.cs

```diff
@@ -30,11 +34,18 @@ public class ReleaseNotesBuilderIntegrationTests
         public void Configure()
         {
             _mapper = AutoMapperConfiguration.Configure();
-            Log.Logger = new LoggerConfiguration().WriteTo.Console().CreateLogger();
+            _logger = new LoggerConfiguration().WriteTo.Console().CreateLogger();
+            Log.Logger = _logger;
 
             _username = Environment.GetEnvironmentVariable("GITTOOLS_GITHUB_USERNAME");
             _password = Environment.GetEnvironmentVariable("GITTOOLS_GITHUB_PASSWORD");
             _token = Environment.GetEnvironmentVariable("GITTOOLS_GITHUB_TOKEN");
+
+            var credentials = string.IsNullOrWhiteSpace(_token)
+                ? new Credentials(_username, _password)
+                : new Credentials(_token);
```

Comment:
These three lines are the reason for the failure.
It is the `Credentials` class that is throwing the error, since these weren't available/used in your previous PR you didn't get any error here.
Also, the integration tests themself are actually marked as explicit, meaning they will only run when they are explicitly initiated.

Reply:
Ahh, I see! I have extracted the lines from the `GitHubProvider`, which is in the `Test`methods and they are marked as `Explicit`. I will try to set the attribute to the class. You helped me a lot! Thank you very much!